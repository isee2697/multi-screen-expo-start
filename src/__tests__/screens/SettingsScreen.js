import React from 'react';
import renderer from 'react-test-renderer';

import SettingsScreen from '../../screens/SettingsScreen';

const navigation = {
  goBack: jest.fn(),
  navigate: jest.fn()
};

const tree = renderer
  .create(<SettingsScreen navigation={navigation} />)
  .toJSON();

describe('<SettingsScreen />', () => {
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
