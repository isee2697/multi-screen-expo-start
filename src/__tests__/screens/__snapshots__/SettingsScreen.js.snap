// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsScreen /> renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <ScrollViewMock
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "paddingTop": 30,
      }
    }
  >
    <Component
      style={
        Object {
          "color": "#1d2d3a",
        }
      }
    >
      Settings Content Area
    </Component>
    <Component
      style={
        Object {
          "fontFamily": "pacifico",
          "fontSize": 20,
          "marginTop": 16,
        }
      }
    >
      Pacifico Font Example
    </Component>
  </ScrollViewMock>
</Component>
`;
